hca/dss/util/__init__.py

```diff
@@ -1,3 +1,45 @@
+import os
+
+
 def separator_to_camel_case(separated, separator):
     components = separated.split(separator)
     return "".join(x.title() for x in components)
+
+
+def directory_builder(src_dir: str):
```

Comment:
Typing (`: str`) is python 3 only and this repo needs to be py2+3.

Reply:
:(  alright, changing, thanks!